joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstCreator.scala

```diff
@@ -1079,7 +1086,7 @@ class AstCreator(filename: String, phpAst: PhpFile, global: Global) extends AstC
       case None =>
         // With variable variables, it's possible to use a valid variable without having an obvious assignment to it.
         // If a name is unknown at this point, assume it's a local that had a value assigned in some way at some point.
-        val local = NewLocal().name(identifier.name).code(s"$$${identifier.code}").typeFullName(identifier.typeFullName)
+        val local = NewLocal().name(identifier.name).code(s"$$${identifier.code}")
```

Comment:
Why are we no longer setting a type here? (Not saying it's wrong, genuinely wondering.)

Reply:
That's a good catch. There should be something there, so I suspect I removed that to replace it with something else (since we'd need to check that the typeFullName exists or use ANY otherwise) and forgot to do the replacement. I'll go through the PR to make sure I didn't miss another.